ENIGMAsystem/SHELL/Platforms/SDL/Gamepad.cpp

```diff
@@ -7,6 +7,13 @@ namespace enigma {
 
 static std::vector<Gamepad> gamepads;
 
+using namespace enigma_user;
+
+int digitalButtons[15] = {
```

Comment:
15 is a magic number. Ishould be a gpad_last_btn or whatever

Reply:
YoYoGames is weird because `gp_axisrv` is `32788` in GMSv1.4. They don't just make simple enums like ENIGMA where `19` is the value. Even for SDL `15` is the value of `SDL_CONTROLLER_BUTTON_MAX`.

Reply:
That sounds like the correct variable to use here then. This is a map of 15 buttons or whatever. The values in the map are unrelated to the number of keys